<launch>
    <include file="$(find ar_marker)/launch/turtlebot3_marker.launch"/>
    <include file="$(find open_manipulator_controller)/launch/open_manipulator_controller.launch"/>
    <include file="$(find open_manipulator_ar_markers)/launch/ar_pose.launch">
        <arg name="camera_model" default="realsense_d435" />
    </include>
    
	<node pkg="simple_navigation_goals" type="pub_ar_pose.py" name="ar_pose" respawn="true" />
	<node pkg="simple_navigation_goals" type="pub_tb3_pose2d.py" name="tb3_pose2d" respawn="true" />
	<node pkg="simple_navigation_goals" type="pub_scan_float32.py" name="scan_float32" respawn="true" />
	
	
	<node pkg="simple_navigation_goals" type="step1_simple_navigation.py" name="step1_nav_arrive" output="screen" >
	    <param name="nav_arrive_end" type="bool" value="false" />
	</node>
	
	<node pkg="simple_navigation_goals" type="step2_move_marker.py" name="step2_marker_arrive" output="screen" >
	    <param name="marker_arrive_end" value="false" />
	</node>
	
	<node pkg="simple_navigation_goals" type="step3_move_scan_data.py" name="step3_wall_arrive" output="screen" />
    
    <param name="wall_arrive_end" value="false" />
	
	<include file="$(find open_manipulator_pick_and_place)/launch/open_manipulator_pick_and_place.launch">
	
</launch>
